automation/post_discussion.py

```diff
@@ -1,5 +1,23 @@
 import huggingface_hub as hf_hub
+
+def construct_post_content(report):
+    # Construct the content of the post
+    opening = "Hey Team!🤗✨ <br />We’re thrilled to share some amazing evaluation results that’ll make your day!🎉📊<br />"
+    disclaimer = "\n\n**Disclaimer**: it's important to note that automated scans may produce false positives or miss certain vulnerabilities. We encourage you to review the findings and assess the impact accordingly.\n"
+    whatsnext = "\n### 💡 What's Next?\nThe Giskard community is always buzzing with ideas. 🐢🤔 What do you want to see next? Your feedback is our favorite fuel, so drop your thoughts in the community forum! 🗣️💬 Together, we're building something extraordinary.\n"
```

Comment:
I think we need to post a link to https://huggingface.co/spaces/giskardai/giskard to promote Giskard hub.

And post the link to the uploaded project, once it's done.